### PR TITLE
Crates publishing updates

### PR DESCRIPTION
**Pull request by rbuysse**

### CODE REVIEW

File: ci/sawtooth-publish-xo-crates [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    libssl-dev \
+    libzmq3-dev \
+    pkg-config \
+    unzip \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+# For Building Protobufs
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y \
+ && curl -OLsS https://github.com/google/protobuf/releases/download/v3.5.1/protoc-3.5.1-linux-x86_64.zip \
```
Comment (vaporos) [inline ci/sawtooth-publish-xo-crates]: Ancient version of protobuf.

File: examples/xo_rust/Cargo.toml [hunk 2]
```diff
@@ -15,9 +15,14 @@
 
 [package]
 name = "sawtooth-xo"
-version = "0.8.0"
+version = "0.3.1"
 authors = ["Bitwise IO, Inc."]
 edition = "2018"
+license = "Apache-2.0"
+description = """\
+    Sawtooth XO is a Sawtooth Transaction Handler for executing Tic-tac-toe \
```
Comment (vaporos) [inline examples/xo_rust/Cargo.toml]: Xo not XO

File: examples/xo_rust/Cargo.toml [hunk 3]
```diff
@@ -15,9 +15,14 @@
...
+    (or Naughts and Crosses) transactions as part of a Sawtooth Distributed ledger
```
Comment (vaporos) [inline examples/xo_rust/Cargo.toml]: Hyperledger Sawtooth instead of "a Sawtooth Distributed ledger"

File: ci/sawtooth-publish-xo-crates (hunk 1 above)
Comment (vaporos) [reply ci/sawtooth-publish-xo-crates]: Should really do what we do in grid:

# Install Rust
RUN curl https://sh.rustup.rs -sSf > /usr/bin/rustup-init \
 && chmod +x /usr/bin/rustup-init \
 && rustup-init -y

# Install protoc
RUN curl -OLsS https://github.com/google/protobuf/releases/download/v3.7.1/protoc-3.7.1-linux-x86_64.zip \
    && unzip -o protoc-3.7.1-linux-x86_64.zip -d /usr/local \
    && rm protoc-3.7.1-linux-x86_64.zip

File: ci/sawtooth-publish-xo-crates [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+ && unzip protoc-3.5.1-linux-x86_64.zip -d protoc3 \
+ && rm protoc-3.5.1-linux-x86_64.zip
+
+ENV PATH=$PATH:/protoc3/bin:/root/.cargo/bin
```
Comment (vaporos) [inline ci/sawtooth-publish-xo-crates]: If you do the above, this turns into "ENV PATH=$PATH:/root/.cargo/bin"

File: examples/xo_rust/Cargo.toml (hunk 2 above)
Comment (peterschwarz) [reply examples/xo_rust/Cargo.toml]: It's "XO" according to the transaction family spec: https://sawtooth.hyperledger.org/docs/core/nightly/master/transaction_family_specifications/xo_transaction_family.html

Comment (chenette) [reply examples/xo_rust/Cargo.toml]: No initial caps for:
transaction handler 
tic-tac-toe 
noughts and crosses  (note the spelling correction)

All caps for:
XO

**Pull request merged**